setup.py

```diff
@@ -15,9 +15,9 @@
     include_package_data=True,
     install_requires=[
         "playwright>=1.18",
-        "pytest",
-        "pytest-base-url",
-        "python-slugify",
+        "pytest>=6.0.0,<8.0.0",
+        "pytest-base-url>=1.0.0,<3.0.0",
+        "python-slugify>=6.0.0,<7.0.0",
```

Comment:
How did you decide on these version ranges?

Reply:
Updated the Pytest version range.

The new values are the minimum which make the test-suite pass: `pip install pytest-base-url==1.0.0 python-slugify==6.0.0 pytest==6.2.0`